### ISSUE TITLE
The readme link at gradle.org is broken

### ISSUE THREAD

**Issue by PerMildner** [bmuschko/gradle-docker-plugin#856]
The page https://plugins.gradle.org/plugin/com.bmuschko.docker-remote-api has a link to https://github.com/bmuschko/gradle-docker-plugin/blob/master/README.asciidoc but that link gives a 404 error.

Possibly related to #567

**Comment by bmuschko** [comment]
> The page https://plugins.gradle.org/plugin/com.bmuschko.docker-remote-api has a link to https://github.com/bmuschko/gradle-docker-plugin/blob/master/README.asciidoc but that link gives a 404 error.

I changed the publishing code a while ago to use a different URL but it looks like the information does not get updated on the plugin portal. The plugin portal still uses the information I used when publishing one of the first versions. I'll also the Gradle team to fix it.

> Possibly related to #567

Those are two different issues. The one you linked to is the user guide of the plugin we publish. The link on the Gradle plugin portal is mostly controlled by Gradle Inc.
